src/containers/pages/FocusInvestigation/single/index.tsx

```diff
@@ -215,13 +215,13 @@ class SingleFI extends React.Component<RouteComponentProps<RouteParams> & Single
     };
     const namePaths =
       planById.jurisdiction_name_path instanceof Array ? planById.jurisdiction_name_path : [];
-    const pages = namePaths.map(namePath =>
+    const pages = namePaths.map((namePath, i) => {
       // return a page object for each name path
-      ({
+      return {
         label: namePath,
-        url: '',
-      })
-    );
+        url: FI_URL + '/filter/' + planById.jurisdiction_path[i] + '/' + planById.id,
```

Comment:
Please don't use string concatenation here. Check elsewhere in the code for what we have been consistently using.